homeworks/igor.orel_igor0806/GamesProject/form.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta
+            name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        >
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Form</title>
+        <link rel="stylesheet" href="form.css">
+    </head>
+    <body>
+
+        <!-- Header  -->
+        <header class="intro">
+            <div class="nav">
+                <div class="logo">
+                    <img src="icon/logo.svg" alt="img">
+                </div>
+
+                <div class="nav-menu">
+                    <ul class="link_list">
+
+                        <li class="link">
+                            <a class="nav-link" href="./games.html">Games</a>
+                        </li>
+
+                        <li class="link">
+                            <a class="nav-link" href="./index.html">About</a>
+                        </li>
+
+                        <li class="link">
+                            <a class="nav-link" href="./form.css">Login</a>
+                        </li>
+
+                    </ul>
+                </div>
+            </div>
+        </header>
+
+        <h1 class="login_title">Login page</h1>
+        <div class="form_centr">
+
+            <form class="login_form" action="./succes.html">
+                <label for="email">Email</label>
+                <br>
+                <input type="email" class="login_input" id="email" name="email" placeholder="Enter your email address" required>
+                <br>
+                <label for="password">Password</label>
+                <br>
+                <input type="password" class="login_input" id="password" name="password" placeholder="Enter your password" required="">
+                <br>
+                <input type="checkbox" id="robot" name="not_a_robot" required="">
+                <label for="robot">I’m not a robot</label>
+                <br>
+                <input type="submit" value="Login">
+            </form>
+        </div>
+
+        <footer class="footer">
+            <div class="social">
+                <a href="#">
+                    <img src="icon/facebook.svg" alt="img">
+                </a>
+
+                <a href="#">
+                    <img src="icon/in.svg" alt="img">
+                </a>
+
+                <a href="#">
+                    <img src="icon/insta.svg" alt="img">
```

Comment:
It's better to save SVG icons and use them

Reply:
Hi, maybe I misunderstood, but I’m already using the SVG format

Reply:
I mean the tag ```svg```
It should looks like this code on the page
```<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M18.9 0H1.1C0.808262 0 0.528473 0.115893 0.322183 0.322183C0.115893 0.528473 0 0.808262 0 1.1V18.9C0 19.1917 0.115893 19.4715 0.322183 19.6778C0.528473 19.8841 0.808262 20 1.1 20H10.68V12.25H8.08V9.25H10.68V7C10.6261 6.47176 10.6885 5.93813 10.8627 5.43654C11.0369 4.93495 11.3188 4.47755 11.6885 4.09641C12.0582 3.71528 12.5068 3.41964 13.0028 3.23024C13.4989 3.04083 14.0304 2.96225 14.56 3C15.3383 2.99521 16.1163 3.03528 16.89 3.12V5.82H15.3C14.04 5.82 13.8 6.42 13.8 7.29V9.22H16.8L16.41 12.22H13.8V20H18.9C19.0445 20 19.1875 19.9715 19.321 19.9163C19.4544 19.861 19.5757 19.78 19.6778 19.6778C19.78 19.5757 19.861 19.4544 19.9163 19.321C19.9715 19.1875 20 19.0445 20 18.9V1.1C20 0.955546 19.9715 0.812506 19.9163 0.679048C19.861 0.54559 19.78 0.424327 19.6778 0.322183C19.5757 0.220038 19.4544 0.139013 19.321 0.0837326C19.1875 0.0284524 19.0445 0 18.9 0Z" fill="#221F1F"/>
</svg>
```